sherlock/resources/data.json

```diff
@@ -285,6 +285,14 @@
     "username_claimed": "blue",
     "username_unclaimed": "noonewouldeverusethis7"
   },
+  "ChessKid": {
+    "errorMsg": "",
```

Comment:
Because of how `message` errorType is implemented (https://github.com/sherlock-project/sherlock/blob/master/sherlock/sherlock.py#L331) such `errorMsg` won't work.